db/schema.rb

```diff
@@ -22,6 +22,7 @@
     t.integer "quantity"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "status", default: 0
```

Comment:
Since there were only 2 options (unfulfilled/fulfilled), maybe this is another place where we could have used a boolean instead (like with the merchant enabled/disabled function).

Then again, maybe the enum is more helpful because it specifies the type of function (instead of just `active?` being true/false).

Thinking out loud...